src/libos/src/time/mod.rs

```diff
@@ -252,3 +252,12 @@ impl TimeProvider for OcclumTimeProvider {
         }
     }
 }
+
+// For Timerfd
+#[repr(C)]
+#[derive(Debug, Default, Copy, Clone)]
+#[allow(non_camel_case_types)]
+pub struct itimerspec_t {
+    it_interval: timespec_t,
+    it_value: timespec_t,
+}
```

Comment:
Should implement  a validate function for itimerspec_t. I think just validate it_interval and it_value is ok

Reply:
Done